src/permission-set.js

```diff
@@ -410,18 +410,23 @@ class PermissionSet {
       debug('Individual access granted for ' + resourceUrl)
       return Promise.resolve(true)
     }
-    // If there are no group authorizations, no need to proceed
-    if (!this.hasGroups()) {
-      debug('No groups authorizations exist')
-      return Promise.resolve(false)
+
+    if (this.hasGroups()) {
+      // Lastly, load the remote group listings, and check for group auth
+      debug('Check groups authorizations')
+
+      return this.loadGroups(options)
+	    .then(() => {
+		return this.checkGroupAccess(resourceUrl, agentId, accessMode, options)
```

Comment:
The indent seems to have changed here.

Reply:
Eh, yeah... My Emacs setup isn't quite there yet...

Reply:
We might want an https://editorconfig.org/ for all Solid projects. Really useful.